orchestration/src/test/java/com/sap/ai/sdk/orchestration/spring/OrchestrationChatResponseTest.java

```diff
@@ -20,11 +20,11 @@ void testToAssistantMessage() {
             .message(ResponseChatMessage.create().role("assistant").content("Hello, world!"))
             .finishReason("stop");
 
-    AssistantMessage message = OrchestrationSpringChatResponse.toAssistantMessage(choice);
+    Generation generation = OrchestrationSpringChatResponse.toGeneration(choice);
```

Comment:
(Minor)

By changing this method call, you're making the test method name inconsistent with what's happening.
Are you sure, you don't want to create a new test case?

Reply:
I renamed the test